src/main/java/it/gov/pagopa/pu/bff/connector/registries/client/DebtPositionRegistrySearchClient.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.pu.bff.connector.registries.client;
+
+import it.gov.pagopa.pu.bff.connector.registries.config.RegistriesApisHolder;
+import it.gov.pagopa.pu.registries.dto.generated.CollectionModelDebtPositionRegistry;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+
+@Service
+@Slf4j
+public class DebtPositionRegistrySearchClient {
+
+  private final RegistriesApisHolder registriesApisHolder;
+
+  public DebtPositionRegistrySearchClient(
+    RegistriesApisHolder registriesApisHolder) {
+    this.registriesApisHolder = registriesApisHolder;
+  }
+
+  public CollectionModelDebtPositionRegistry findDebtPositionRegistries(
+    Long debtPositionId, String accessToken) {
+    try{
+      return registriesApisHolder.getDebtPositionRegistrySearchControllerApi(
+          accessToken)
+              .crudDebtPositionRegistriesFindAllByDebtPositionId(debtPositionId);
+    } catch (HttpClientErrorException.NotFound e) {
+      log.warn("DebtPositionRegistries with debtPositionId {} not found", debtPositionId);
+      return null;
+    }
```

Comment:
The APIs which return a collection cannot return 404!
404 would means this way an API not found, let's not handle this ;)